### PR TITLE
SPOI-8789 Adding example for fixed width parser

### PR DESCRIPTION
**Pull request by KapoorHitesh**
@chinmaykolhatkar please review and merge. it is rebased

### CODE REVIEW

File: tutorials/parser/pom.xml [hunk 1]
```diff
@@ -275,6 +275,18 @@
       <optional>true</optional>
     </dependency>
     <dependency>
+      <groupId>com.univocity</groupId>
+      <artifactId>univocity-parsers</artifactId>
+      <version>2.0.0</version>
+      <optional>true</optional>
```
Comment (chinmaykolhatkar) [inline tutorials/parser/pom.xml]: Why is this optional true?

File: tutorials/parser/src/test/java/com/datatorrent/tutorial/fixedwidthparser/fixedWidthParserApplicationTest.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              String fileData = FileUtils.readFileToString(parsedFile);
+              String[] campaigns = fileData.split("\n");
+              return campaigns.length == 10;
+            }
+          }
+          return false;
+        }
+      });
+
+      lc.run(30 * 1000); // runs for 30 seconds and quits
```
Comment (chinmaykolhatkar) [inline tutorials/parser/src/test/java/com/datatorrent/tutorial/fixedwidthparser/fixedWidthParserApplicationTest.java]: If you've set an exit condition why should it be required to run for specific interval?

**Review by chinmaykolhatkar — CHANGES_REQUESTED** [review]
Minor changes.. Otherwise looks good.

File: tutorials/parser/src/test/java/com/datatorrent/tutorial/fixedwidthparser/fixedWidthParserApplicationTest.java (hunk 2 above)
Comment (Hitesh-Scorpio) [inline tutorials/parser/src/test/java/com/datatorrent/tutorial/fixedwidthparser/fixedWidthParserApplicationTest.java]: If the exit condition never meets, due to some bug or failure or any reason, still then the application should quit after a specific interval. Similar approach has been used in other test apps.

File: tutorials/parser/pom.xml (hunk 1 above)
Comment (Hitesh-Scorpio) [inline tutorials/parser/pom.xml]: This dependency is only used for certain features in the project, and will not be needed if that feature isn't used. (On the lines of super-csv dependency)

**Review by chinmaykolhatkar — APPROVED** [review]

**Pull request merged**